composer.json

```diff
@@ -16,18 +16,21 @@
     "classmap": [
       "src/models/",
       "src/views/",
-      "src/controllers/"
+      "src/controllers/",
+      "src/enums/",
+      "src/exceptions/",
+      "src/interfaces/"
     ]
   },
   "require": {
-    "php": "^7.4|^8.0",
+    "php": "^8.1",
```

Comment:
The `CONTRIBUTING.md` docs will also need to be updated with the new minimum version